### ISSUE TITLE
[Bug]: `--lowest` parameter does not work in CLI

### ISSUE THREAD

**Issue by Willmish** [Green-Software-Foundation/carbon-aware-sdk#97]
### Contact Details

GitHub/Slack: Willmish

### What happened?

The CLI does not change the output to a single, best location and time, but simply omits the parameter (parses it, but doesn't do anything with it), see the diff file with some changes I tried making to fix it (currently CarbonAwareCOnstants.best props is set to true by default, but never used later on - only ` _aggregator.GetEmissionsDataAsync()` is called but never `_aggregator.GetBestEmissionsDataAsync()`

```diff
diff --git a/src/CarbonAware.CLI/src/CarbonAwareCLI.cs b/src/CarbonAware.CLI/src/CarbonAwareCLI.cs
index 350639d..db2ff3d 100644
--- a/src/CarbonAware.CLI/src/CarbonAwareCLI.cs
+++ b/src/CarbonAware.CLI/src/CarbonAwareCLI.cs
@@ -63,16 +63,23 @@ public class CarbonAwareCLI
             { CarbonAwareConstants.MultipleLocations, locations },
             { CarbonAwareConstants.Start, _state.Time },
             { CarbonAwareConstants.End, _state.ToTime },
-            { CarbonAwareConstants.Best, true }
  1 diff --git a/src/CarbonAware.CLI/src/CarbonAwareCLI.cs b/src/CarbonAware.CLI/src/CarbonAwareCLI.cs
+            { CarbonAwareConstants.Best, _state.Lowest }
         };
         return await GetEmissionsDataAsync(props);
     }

     private async Task<IEnumerable<EmissionsData>> GetEmissionsDataAsync(Dictionary<string, object> props)
     {
-        IEnumerable<EmissionsData> e = await _aggregator.GetEmissionsDataAsync(props);
+        if ((bool)props[CarbonAwareConstants.Best])
+        {
+            EmissionsData e = await _aggregator.GetBestEmissionsDataAsync(props);
+            return new IEnumerable<EmissionsData> {e};
+        }
+        else
+        {
+            return await _aggregator.GetEmissionsDataAsync(props);
+        }

-        return await _aggregator.GetEmissionsDataAsync(props);
     }

     public void OutputEmissionsData(IEnumerable<EmissionsData> emissions)
```
     
     

### client

CLI

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danuw** [comment]
We should be returning an array for lowest, as we could get multiple times or regions that have the same lowest you would then should from.

TO BE MERGED with #36

**Comment by Willmish** [comment]
> We should be returning an array for lowest, as we could get multiple times or regions that have the same lowest you would then should from.
> 
> 
> 
> TO BE MERGED with #36 

@danuw this should be changed for the web API as well in that case as it might be currently just returning 1 value as well - the GetBestEmissionsByLocations function - will link out to code once I find it

**Comment by dylHall** [comment]
@Willmish, I'm on it - and should have a code review out soon! Think I've found the code, so feel free to leave a message there

**Comment by vaughanknight** [comment]
Resolved in #104
